src/style.css

```diff
@@ -0,0 +1,84 @@
+/* stylelint-disable max-line-length */
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+
+  --color-white: #fff;
+  --color-silver: #e1e7ed;
+  --color-gray: #3d4e61;
+  --color-dark: rgba(61, 78, 97, 0.1);
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.form__input {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  background: var(--color-white);
+  border: 1px solid var(--color-silver);
+  box-shadow: 0 1px 8px var(--color-dark);
+  border-radius: 4px;
+  outline: none;
+
+  font-family: inherit;
```

Comment:
this is default value, you don't need to mention it explicitly

Reply:
hmm, interesting

Reply:
Now I cant to push my commit. 
![commit](https://user-images.githubusercontent.com/109144184/219959723-9ead0722-44cb-47c7-9e96-e50f74b6f6c6.jpg)

Reply:
Ok, I did it.
Must use git push -f origin develop.

Reply:
do `git pull origin develop` first in cases like that. and ask your questions in the Slack next time please, you'll get your answers faster